.github/workflows/jdk-early-access-build.yml

```diff
@@ -10,7 +10,7 @@ on:
         description: 'JDK version'
         required: true
         # make sure to keep the default of JDK_VERSION in sync!
-        default: '17-ea'
+        default: '16'
```

Comment:
Is PR still in WIP mode? I'm asking because the EA job moved on to 17-ea already and should not be rolled back like this.
What you can (and should) do though is trigger this job manually with `16` in your fork. No need to change the file.

Reply:
Oh no, I misunderstood your comment I will revert that

Reply:
No worries, it's a monday!